Automattic/pocket-casts-android
# Merge main and bump horologist to 0.2.7

## Description

This PR
- Merges latest main to add/wear (7150e630f013320bca2188360654334f3b56a7f8...68608b3fdb0fc520acff7a17c3a7b8d435782ef9)
- Bumps horologist from 0.1.11 to [0.2.7](https://github.com/google/horologist/releases/tag/v0.2.7), replaces deprecated methods (74290bd1f2ca32aa4d2a11e5f658c1b54d36b443...52ce380942347dd472fca9d162efbf2da944e174)

## Testing Instructions

- Make sure that the wear app compiles and installs fine on the watch emulator/ device
- Smoke test that you can see the home screen and sign-in to see your up-next list
